docs/parameters.md

```diff
@@ -25,22 +28,25 @@ Parameters a user can specify when provisioning an FSx OpenZFS file system:
+| `DailyAutomaticBackupStartTime` | No        | `'"00:00"`                                                                                                                                                                                                                                                                               |                                                                                                                                                      |
+| `DiskIopsConfiguration`         | No        | `'{"Iops": 300, "Mode": "USER_PROVISIONED"}'`                                                                                                                                                                                                                                            |                                                                                                                                                      |
+| `EndpointIpAddressRange`        | No        | `'"192.168.0.0/24"'`                                                                                                                                                                                                                                                                     |                                                                                                                                                      |
+| `PreferredSubnetId`             | MAZ       | `'"subnet-016affca9638a1e61"'`                                                                                                                                                                                                                                                           |                                                                                                                                                      |
+| `RootVolumeConfiguration`       | No        | `'{"CopyTagsToSnapshots": false, "DataCompressionType": "NONE", "NfsExports": [{"ClientConfigurations": [{"Clients": "*", "Options": ["rw","crossmnt"]}]}], "ReadOnly": false, "RecordSizeKiB": 128, "UserAndGroupQuotas": [{"Type": "USER", "Id": 1, "StorageCapacityQuotaGiB": 10}]}'` |                                                                                                                                                      |
+| `RouteTableIds`                 | No        | `'["rtb-01e1c21b8peb65a3f"]'`                                                                                                                                                                                                                                                            |                                                                                                                                                      |
```

Comment:
Should we group the MAZ-specific parameters? I don't feel too strongly about this, but maybe it'd be easier to understand.

Reply:
I added a comment denoting what parameters for for multi-az.